unity-renderer/Assets/DCLPlugins/ECS7/ComponentWrapper/Generic/WrappedComponentPool.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+
+namespace DCL.ECS7.ComponentWrapper.Generic
+{
+    public record WrappedComponentPool<T> where T: class, IWrappedComponent
+    {
+        private readonly Queue<PooledWrappedComponent<T>> queue;
```

Comment:
+1, we are using the built-in pool already

Reply:
I would love to, sadly the built-in pool does not satisfy the needs here, since the pooled objects are reference to a interface of methods to serialize component, so there is no specific type (it could be any component type), so I couldn't actually release any object because the actual type would be unknown.
There is a `PooledObject<T>` in the built-in pool class, but unfortunately it's impossible to get the value from that object since is private 😢